e2e/app.e2e.ts

```diff
@@ -84,6 +84,46 @@ describe('bterm launch', function() {
         .then(() => this.app.client.browserWindow.isFocused('.theme-browser > span:nth-child(2)'))
```

Comment:
You are right. Correct way of checkin this is ` this.app.client.hasFocus('.theme-browser > span:nth-child(2)'))` Thanks.